apps/ivrs/api_views.py

```diff
@@ -49,14 +50,20 @@ def get(self, request):
 
         is_invalid = True
 
-        if not is_user_registered(parameters['telephone']):
+	if parameters['telephone'] == None:
+            message = get_message(parameters, is_user_registered=False)
+
+        elif not is_user_registered(parameters['telephone']):
             message = get_message(parameters, is_user_registered=False)
 
         elif not is_data_valid(processed_data):
             message = get_message(parameters, is_data_valid=False)
 
         elif not is_logically_correct(processed_data):
             message = get_message(parameters, is_logically_correct=False)
+        
+        elif not school_id.isdigit():
+           message = get_message(parameters, is_school_id_valid=False)
```

Comment:
Ah okay. Then could you add this check within that function itself? Best to keep the view abstract and hide away the implementation details within the functions.